Doc/library/reprlib.rst

```diff
@@ -123,6 +123,16 @@ which format specific object types.
    similar manner as :attr:`maxstring`.  The default is ``20``.
 
 
+.. attribute:: Repr.indent
+
+   If this attribute is set to ``None`` (default), no indentation of the output
```

Comment:
These docs are quite confusing and have to be read multiple times to understand, even though the API itself is more or less intuitive.

Reply:
Thank you. I appreciate the review and feedback. Unfortunately, I'm not sure how to improve the docs paragraph. I've asked for help here: https://discuss.python.org/t/hint-or-advice-how-to-improve-docs-about-reprlib-repr-indent-feature/15747

Reply:
@Ovsyanka83 Do the suggested changes below resolve your request for improved docs?